lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -14,20 +14,21 @@ $-logo-height: 34px;
 $-left-right-spacing: var(--nx-spacing-6x);
 
 $-text-color: var(--nx-swatch-white);
-$-background-color: var(--nx-swatch-blue-10);
-
-$-active-color: var(--nx-swatch-indigo-50);
-$-focus-color: var(--nx-swatch-blue-95);
-$-hover-color: var(--nx-swatch-indigo-30);
 
 $-button-border-width: 2px;
 
-$-separator-color: var(--nx-swatch-indigo-40);
+@mixin link-no-outline {
```

Comment:
By using a mixin for this, it's going to get repeated in the stylesheet three times.  Better to use `@extend` and a placeholder selector, or simply write it directly in a `&:hover, &:active, &.selected` block

Reply:
> By using a mixin for this, it's going to get repeated in the stylesheet three times. Better to use `@extend` and a placeholder selector, or simply write it directly in a `&:hover, &:active, &.selected` block

Removed mixin: fcf2ac5174c3a5e7a1545a99fb28f18cb2f8931d
Combined interaction states: c091bdf7ae549f39a5f431479de68641acfe8cd0